lib/con_cache.ex

```diff
@@ -163,10 +165,12 @@ defmodule ConCache do
     options =
       Keyword.merge(options, ttl: options[:global_ttl], ttl_check: options[:ttl_check_interval])
 
+    n_lock_partitions = Keyword.get(options, :n_lock_partitions, System.schedulers_online())
```

Comment:
We should also add the corresponding typespec entry to the `options` type (line 76).